src/store/reducers/shardsWorkload/shardsWorkload.ts

```diff
@@ -128,12 +131,12 @@ export const shardApi = api.injectEndpoints({
                         {
                             query:
                                 filters?.mode === EShardsWorkloadMode.Immediate
-                                    ? createShardQueryImmediate(path, sortOrder, database)
+                                    ? createShardQueryImmediate(path, database, sortOrder)
                                     : createShardQueryHistorical(
                                           path,
+                                          database,
```

Comment:
As I see here path and database can exist simulteneously
Could you please explain the real difference (as somewhere we change path to database and somewhere not)

Reply:
Example: we look at shards of a table `db1/table1`
Database is `db1`, it should be used as a request param, system tables in `db1/.sys/`, but the result should be filtered, so only shards of current path `db1/table1` will be shown

Reply:
`path` - path to schema object
`database` - database, where this schema object exist